### DISCUSSION TITLE
Search for math symbols using either a Unicode character or its code point (U+...) as the input in the documentation's symbols reference page

### DISCUSSION THREAD

**Discussion by floristfromnowhere** [typst/typst#452] [Ideas]
I was trying to look for the Typst symbol name for the Unicode character ◅ (U+025C5) (or `\whitepointerleft` in `LaTeX`) but I couldn't find it on [the documentation's symbols reference page](https://typst.app/docs/reference/symbols/sym/). It would be a nice feature to be able to search for math symbols using either a Unicode character or its code point (U+...) as the input. (For example, there's [this website for converting a Unicode character to a `LaTeX` expression](https://www.johndcook.com/unicode_latex.html).)

**Comment by Enivex** [comment]
Not really an answer to you question, but you can enter unicode symbols with \u{} if you know the code point.

**Reply by laurmaedje** [reply]
Searching with the Unicode character is already possible. `\whitepointeleft` simply has no analogue in Typst at the moment. Searching with the codepoint seems like a worthwhile addition.
